ui-frontend/packages/catalog-ui-search/src/main/webapp/component/tabs/metacard/summary.tsx

```diff
@@ -36,6 +36,7 @@ import { displayHighlightedAttrInFull } from './highlightUtil'
 import DateTimePicker from '../../fields/date-time-picker'
 import Geometry from '../../../react-component/input-wrappers/geometry'
 import { useRerenderOnBackboneSync } from '../../../js/model/LazyQueryResult/hooks'
+import { truncate } from 'lodash'
```

Comment:
Is this used?

Reply:
No idea how it got in there, but nope. Removed.